apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -225,3 +254,115 @@ function PopoverEndDate() {
     </p>
   `.toString();
 }
+
+function CreateCourseInstanceModal({
+  courseShortName,
+  csrfToken,
+  initialStartDateFormatted,
+  initialEndDateFormatted,
+  timezone,
+}: {
+  courseShortName: string;
+  csrfToken: string;
+  initialStartDateFormatted: string;
+  initialEndDateFormatted: string;
+  timezone: string;
+}) {
+  return Modal({
+    id: 'createCourseInstanceModal',
+    title: 'Create course instance',
+    formMethod: 'POST',
+    body: html`
+      <div class="form-group">
+        <label for="long_name">Long name</label>
+        <input
+          type="text"
+          class="form-control"
+          id="long_name"
+          name="long_name"
+          required
+          aria-describedby="long_name_help"
+        />
+        <small id="long_name_help" class="form-text text-muted">
+          The full course instance name, such as "Fall 2025". Users see it joined to the course
+          name, e.g. "${courseShortName} Fall 2025".
+        </small>
+      </div>
+      <div class="form-group">
+        <label for="short_name">Short name</label>
+        <input
+          type="text"
+          class="form-control"
+          id="short_name"
+          name="short_name"
+          required
+          pattern="[\\-A-Za-z0-9_\\/]+"
+          aria-describedby="short_name_help"
+        />
+        <small id="short_name_help" class="form-text text-muted">
+          A short name, such as "Fa25" or "W25b". This is used in menus and headers where a short
+          description is required.
+        </small>
+      </div>
+      <div class="form-check mb-3">
+        <input
+          type="checkbox"
+          class="form-check-input"
+          id="access_dates_enabled"
+          name="access_dates_enabled"
+          aria-describedby="access_dates_enabled_help"
+        />
+        <label class="form-check-label" for="access_dates_enabled">
+          Make course instance available to students
+        </label>
+        <small id="access_dates_enabled_help" class="form-text text-muted">
+          This can be enabled later.
```

Comment:
Nit: do you think this might look better on a separate line?

<img width="371" alt="Screenshot 2024-12-16 at 14 29 13" src="https://github.com/user-attachments/assets/5fc74b87-0d39-4c8a-8f8d-13d2548d14f7" />

If we do this, we can probably also drop the conditional show/hide and just always show this, as indeed student access can be changed (removed entirely) later.

Reply:
I agree, for consistency with the other input fields. 

Here's what it looks like now:

<img width="407" alt="image" src="https://github.com/user-attachments/assets/15d7117b-ee1a-432c-973e-6f364b4bc0e7" />

The "this can be enabled later." text is also always shown:

<img width="497" alt="image" src="https://github.com/user-attachments/assets/7f88011f-cded-48fa-9171-8cd7d504abbf" />

Reply:
Looks great 💯